### DISCUSSION TITLE
Export von Zählerdaten z.B. per MQTT

### DISCUSSION THREAD

**Discussion by Lelek62** [camueller/SmartApplianceEnabler#589] [Q&A] [answered]
Gibt es sowas schon? Ich würde gern S0 Zählerdaten an openHAB, z.B. per MQTT weitergeben.
Gruß
Erik

**Comment by camueller** [comment, marked as answer]
Einfach in die Doku schauen: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/MQTT_DE.md

**Reply by Lelek62** [reply]
Danke, hatte ich gesucht aber tatsächlich nicht gefunden. Aber super das das geht!
